text/0034-precomputed-entity-attributes.md

```diff
@@ -2,11 +2,11 @@
 
 ## Related issues and PRs
 
-- This RFC is mostly directly taken from [RFC 28](https://github.com/cedar-policy/rfcs/pull/28);
+- Related RFC(s): This RFC is mostly directly taken from [RFC 28](https://github.com/cedar-policy/rfcs/pull/28);
```

Comment:
Wouldn't it be nicer to link to the Markdown file associated to that RFC? That's going to be the most up-to-date version of the RFC. Same goes for other RFC links in this PR.